data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,835 @@
+				decline
+			label "depart"
+			`	Whatever is happening isn't worth your time.`
+				decline
+
+
+
+mission "Hai Reveal: Media Retraction"
+	landing
+	invisible
 	to offer
-		has "event: hr: media retraction"
+		has "event: hr: exploration period"
 	on offer
+		set "hr: media retraction"
+		event "hr: media retraction reaches hai" 25
 	# Sync reputations for aliases of governments:
 		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
 		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
-		# Make sure the new governments start with appropriate rep:
+	# Make sure the new governments start with appropriate rep:
 		"reputation: Elenctic Commune" = 1000
+
+event "hr: media retraction reaches hai"
+
+
+mission "Hai Reveal: Media Retraction (Human)"
+	landing
+	invisible
+	source
+		government "Republic" "Syndicate" "Free Worlds" "Neutral"
+		attributes "spaceport"
+	to offer
+		has "hr: media retraction"
+		not "Hai Reveal: Media Retraction (Hai): offered"
+	on offer
 		conversation
-			`News report that says that the guy who showed the video of the Hai ships five months prior was just a viral marketing stunt for an upcoming movie.`
+			`One of the usual short news segments between programming, typically present in most significant spaceports, brings up a familiar-looking broadcast. The logo on the bottom is the same program that originally broadcast the news of the Hai. In the short snippet the speaker is issuing a sincere apology for their promotion of a hoax, advising that the video of the Hai ships released five months prior was in fact nothing more than a viral marketing stunt for an upcoming movie.`
+			`	Afterwards, the snippet gives way to a teaser trailer that has a number of pretty combat scenes and says absolutely nothing.`
 				decline
 
 
+mission "Hai Reveal: Media Retraction (Hai)"
+	landing
+	invisible
+	source
+		government "Hai"
+		attributes "spaceport"
+	to offer
+		has "event: hr: media retraction reaches hai"
+	on offer
+		event "hr: pirates respond to leaks" 1
+		event "hr: concern period" 35
+		conversation
+			`One of the usual short news segments between programming, typically present in most significant spaceports, brings up a Hai reporting on a familiar-looking human broadcast. The logo on the bottom is the same program that originally broadcast the news of the Hai. In the short snippet the speaker is issuing a sincere apology for their promotion of a hoax, advising that the video of the Hai ships released five months prior was in fact nothing more than a viral marketing stunt for an upcoming movie.`
```

Comment:
I think this should have a branch depending on whether or not you have seen the news report in human space. In the branch where you haven’t, I would remove the word familiar-looking, and in the branch where you have, I would just say it appears to be the human broadcast you saw earlier, then go straight to the safety advisory.

Reply:
When you see it in Hai space, they show a clip from the human broadcast as part of the piece, so you see the same thing either way just in a couple of different contexts.

Reply:
Yeah, but you don’t need to describe the broadcast the same way twice.